guides/source/active_record_migrations.md

```diff
@@ -369,10 +386,99 @@ end
 
 This method creates a `products` table with a column called `name`.
 
+
+#### Associations
+
+If you're creating a table for a model that has an association, you can use the
+`:references` type to create the appropriate column type. For example:
+
+```ruby
+create_table :products do |t|
+  t.references :category
+end
+```
+
+This will create a `category_id` column. Alternatively, you can use `belongs_to`
+as an alias for `references`:
+
+```ruby
+create_table :products do |t|
+  t.belongs_to :category
+end
+```
+
+You can also specify the column type and index creation using the `:polymorphic`
```

Comment:
Useful to link to polymorphic associations here: association_basics.html#polymorphic-associations